.github/workflows/test-report.yml

```diff
@@ -0,0 +1,25 @@
+name: Generate test reports
+
+on:
+  workflow_run:
+    workflows: [ 'Build and run tests' ]
+    types:
+      - completed

```

Comment:
According to the documentation, this workflow will only start running when the `Build and run tests` workflow is in the `master` branch